packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditionsItem.tsx

```diff
@@ -81,6 +132,39 @@ const TermsAndConditionsItem: React.FunctionComponent<
         </span>
         I agree
       </span>
+    </li>
+  )
+}
+
+export function LoadingItem(props: { numLoadingItems: number }) {
```

Comment:
Here's the loading UI definition (mimics the output from the TermsAndConditionsItem